source/db/zh-TW-projects.json

```diff
@@ -1135,7 +1135,7 @@
     "development_stage": "released",
     "description": "POSIX-compliant enterprise cryptographic stacked filesystem for GNU/Linux.",
     "license_url": "https://git.kernel.org/cgit/linux/kernel/git/tyhicks/ecryptfs.git/tree/COPYING",
-    "logo": "geli.png",
+    "logo": "encryptfs.png",
```

Comment:
Would be better to simply remove all these duplicated translation keys and let them inherit from the English JSON, instead of changing all of them. I'd be okay merging as-is though.